package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@nuxt/devalue",
-  "version": "1.2.4",
+  "version": "1.2.5",
```

Comment:
Version bumps will happen in separate PRs/commits

Reply:
Sorry, made a revert commit